readme.txt

```diff
@@ -417,7 +419,6 @@ Initial Release
 * Added: WPML String Translation setting
 * Added: Persian translations
 * Updated: Font Awesome
-* Updated: Spanish, Portugese, Brazilian, French & Polish Translations
```

Comment:
Since it exceeds 300 characters limits, so I just removed this one:

![image](https://github.com/user-attachments/assets/0e65b4f2-61bf-4afa-a754-e6d129d699aa)

Reply:
Maybe you can make it shorter? I don't think we should remove it.

Reply:
We can only have this much of it:
`* Updated: Spanish, Portuguese, Br`

What do you think about this:

Reply:
> What do you think about this:

Agree

Reply:
Done.